src/components/CustomChannelComponent.tsx

```diff
@@ -110,8 +111,15 @@ const Root = styled.div<RootStyleProps>`
 `;
 
 export function CustomChannelComponent() {
-  const { suggestedMessageContent, botId, enableEmojiFeedback } =
-    useConstantState();
+  const {
+    suggestedMessageContent,
+    botId,
+    enableEmojiFeedback,
+    customUserAgentParam,
+  } = useConstantState();
+  const isForSendbirdDashboard =
+    customUserAgentParam &&
+    customUserAgentParam['chat-ai-widget-preview'] === 'True';
```

Comment:
Can you make it into a util function?

Reply:
like this?
```
function isForSendbirdDashboard(data: object | undefined) {
  return (
    data &&
    'chat-ai-widget-preview' in data &&
    data['chat-ai-widget-preview'] === 'True'
  );
}
```
I think redundant because used in only one place.

Reply:
updated!

Reply:
Yes, but I suggested managing them separately as key-value pairs should be maintained as constants.